crates/router/src/connector/adyen/transformers.rs

```diff
@@ -876,6 +901,13 @@ fn get_additional_data(item: &types::PaymentsAuthorizeRouterData) -> Option<Addi
     }
 }
 
+fn get_channel_type(pm_type: &Option<storage_enums::PaymentMethodType>) -> Option<Channel> {
+    match pm_type {
+        Some(storage_enums::PaymentMethodType::GoPay) => Some(Channel::Web),
+        _ => None,
+    }
```

Comment:
```suggestion
 pm_type.as_ref().and_then(|pmt| match pmt {
        storage_enums::PaymentMethodType::GoPay => Some(Channel::Web),
        _ => None,
    })
```
This suggestion is because, if in future any payment method type requires channel, then we have to add in Some(pmt's) with the existing code.